en/appendices/3-6-migration-guide.rst

```diff
@@ -41,6 +41,9 @@ behavior that may affect your application:
   Previously invalid tasks would be silently ignored.
 * CakePHP internals now chain exceptions where possible, allowing root causes of
   errors to be exposed.
+* MySQL connections now default to ``utf8mb4`` as their encoding. If you want to
+  continue using the old default of ``utf8`` update your application's
+  connection information.
```

Comment:
This is what I'd write. `utf8mb3` clearly transports that it is "not just utf8" which I think will help people.

Take the bits you think that are useful:

    MySQL/MariaDB connections now default ``utf8mb4`` (4-byte full unicode) connection encoding.
    This brings MySQL/MariaDB defaults in line with the other supported database adapters.
    
    If you want to continue using the previous default subset of 3-byte encoding, then you need to set
    your datasource's encoding in ``app.php`` to ``utf8mb3`` (``utf8`` before MySQL 5.5.3 / MariaDB 5.5).

Refs:
- https://dev.mysql.com/doc/refman/5.5/en/charset-unicode-utf8mb3.html
- https://mariadb.com/kb/en/library/unicode/

Reply:
Edit: Not sure about DB2 and `This brings MySQL/MariaDB defaults in line with the other supported database adapters.`

Reply:
utf8mb3 is not a 'thing' though. The old encoding was called 'utf8'.